src/clientdlg.cpp

```diff
@@ -858,6 +858,16 @@ void CClientDlg::OnNumClientsChanged ( int iNewNumClients )
     SetMyWindowTitle ( iNewNumClients );
 }
 
+void CClientDlg::OnOpenGeneralSettings()
```

Comment:
I think the slot name needs to change to match what it does -- `OnOpenGeneralSettings` should be `OnOpenAudioNetSettings`.

Reply:
For readability that is a plus.
I have been going on the minimal changes of existing, but that may not be the best way to go.
I will clean this up this afternoon.